codespeed/views.py

```diff
@@ -731,3 +739,239 @@ def add_json_results(request):
     logger.debug("add_json_results: completed")
 
     return HttpResponse("All result data saved successfully", status=202)
+
+
+def django_has_content_type():
+    return (django.VERSION[0] > 1 or
+            (django.VERSION[0] == 1 and django.VERSION[1] >= 6))
+
+
+def validate_results_request(data):
+    """
+    Validates that a result request dictionary has all needed parameters
+
+    It returns a tuple
+        "", False  when no errors where found
+        Error_message, True when there is an error
```

Comment:
I would change the return value to True, and raise a `ValidationError` (from django.core.exceptions) when there is an error.